### ISSUE TITLE
No override file for FieldHint generated

### ISSUE THREAD

**Issue by r12bs** [dubhater/Wobbly#6]
Wobbly v4.

Marking a frame as combed with the P key makes no difference in the output. The generated script doesn't load any override file either.

FrameProps doesn't show the _Combed property either for frames marked as Combed in Wobbly.

**Comment by dubhater** [comment]
Yeah, marking a frame as combed is a bit useless at the moment. It doesn't do anything with the marked frames.

**Comment by dubhater** [comment]
FieldHint doesn't require an override file in VapourSynth. All the matches are transmitted to FieldHint in the Python script.
